test/compareTo.js

```diff
@@ -14,14 +14,6 @@ describe('Compare test', function () {
                     Big(test.args[1])
                 ).toString();
             };
-            if (test.result === 'errorThrown') {
```

Comment:
I know compareToTestCases.json and some other json files does not include any "errorThrown" test cases, but can we keep this part just for consistency with other test codes? I want to be sure that if Java throws, our lib should also throw regardless of the method that is tested.